### ISSUE TITLE
Create a 5-10 minutes Power Pitch from the Ichi Pixel presentation

### ISSUE THREAD

**Issue by NathanLovato** [GDquest/godot-3-presentation#27]
Based on your feedback on https://github.com/GDquest/godot-3-presentation/issues/26

In general, just take care of the things you feel comfortable with and leave the rest to me. Feel free to open issues if you find bugs or areas to improve - I'll do a call for the community to contribute + a video recording of the new presentation to give it some more visibility.

Videos: just link to the footage you would like to put in a slide (e.g. youtube link with starting time) - I'll bring it in and transcode it myself - if you don't have a lot of exp. with that it's going to take you too much time to encode it right - ffmpeg doesn't have the best docs as you know.

Please create a new `power_pitch` folder or something and create the presentation there! You can start with a copy of the ichi-pixel presentation.

## Annotations on your feedback

Let's remove the Deponia slide. I think a list of professional games/businesses that use Godot right now would have more impact.

```
## All-in-One Package
 - Break into dedicated slides for each feature
```

This will not hold in a short pitch presentation. There's only room for 4 or 5 slides at most. I'd focus on:

1. Extend the Engine... with Game Code! 
1. Cinematic Animation
1. Cutting Edge realtime-GI (+principled BSDF material)

And All the Essentials (bullet point list, multiple pics packed in a GridContainer?)

The tilemap workflow isn't great at the moment, I wouldn't showcase it. There's also too much here for a short presentation. 

```
### New All in One Breakthrough Proposal

- Code (with available languages and their features)
  - Shaders
  - Bultin API
- Visual Editor (SceneTree and Node System, basically what currently is in **Great to Learn Game Code**)
  - 2D Editor 
    - UI (**Godot is a Godot Game** + **Complex User Interface**)
    - Animation (really emphasize this :stuck_out_tongue: )
  - 3D Editor
    - Use **Modern 3D Engine** (using some of the media mentioned in previously)
```

```
## Virtual Reality

- Use footage from [Bastiaan Olij Leap Motion?](https://www.youtube.com/watch?v=vumX-Or9J5U)

## Godot 3.0 is Out!

- Cut
```

Let's just have a general slide for this one: "Hundreds of contributors / Fastest growing game engine on GitHub / Full-time core dev team..."

```
## Godot is in Great Shape

- Mention the [popularization of the engine](https://itch.io/games/tag-godot-engine)
- Update data (?)
  - Maybe design a tool to scavenger the data and always keep it up to date? 
```

```
## Overall

- Allow window scaling
- Use update media with most recent content
- Add a "Coming Soon" slide to showcase what is in the bake
```

**Comment by NathanLovato** [comment]
I will work a bit on this project tomorrow morning at the dev meetup. If you get started by then, even if it's unfinished, please up what you have for review - as it's the first time I get you to work on a presentation, I'd like to work incrementally, review your work early on.

**Comment by NathanLovato** [comment]
Closed with #30
